File: vscodejava/Hello/src/a0321/bank/book.java
package a0321.bank;

public class book {
    private String title; // 책추가
    private String author; // 저자
    private int status; // 대출가능여부

    public book(String title, String author, int status) {
        this.title = title;
        this.author = author;
        this.status = status;
    }

    // 책 제목
    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }
    
    // 책 저자
    public String getAuthor() {
        return author;
    }

    public void setAuthor(String author) {
        this.author = author;
    }

    // 책 반납 상태
        public int getStatus() {
        return status;
    }

    public void setStatus(int status) {
        this.status = status;
    }

}

File: Vs_C/a0414/book.cpp
#include <iostream>
#include <string>
using namespace std;

class Book {
private:
    string title;
    string author;
    int price;

public:
    // 생성자
    Book(string t, string a, int p) {
        title = t;
        author = a;
        price = p;
    }

    // 책 정보 출력 함수
    void printInfo() {
        cout << "제목: " << title << "\n";
        cout << "저자: " << author << "\n";
        cout << "가격: " << price << "원" << "\n";
    }
};

int main() {
    Book book1("아아아ㅏ", "나마나ㅏㄴ", 12000);
    book1.printInfo();

    return 0;
}
